apps/site/src/components/dom/SceneLayout.tsx

```diff
@@ -3,7 +3,7 @@
 import { ReactNode, useRef } from "react";
 import Scene from "@/components/canvas/Scene";
 
-const Layout = ({ children }: { children: ReactNode }) => {
+const SceneLayout = ({ children }: { children: ReactNode }) => {
```

Comment:
Nitpick: Since you changed the other instances of `ReactNode` to `PropsWithChildren`, should we do it here too?

Reply:
Uhh, the `Three.tsx` file also uses `ReactNode`, so I didn't bother messing with it too much.